### DISCUSSION TITLE
device freezes after installing lsposed and restarting device

### DISCUSSION THREAD

**Discussion by f9nhcha** [LSPosed/LSPosed#2200] [Q&A]
after i install your module in magisk my device seems to freeze and have to reboot a few times to stabilize and use normally, hope you have a solution for this problem, even though i have installed the module based on riru or zygisk both affected
my device is pixel 3a xl running on pixel experience android 13 and no other module installed
[lsposed.txt](https://github.com/LSPosed/LSPosed/files/9895039/lsposed.txt)

**Comment by leeteral** [comment]
I'm experiencing the same issue here. It's best to delete the LSPosed module to prevent damage such as bootloops.
